docs/planning/adr/adr-001-initial-architecture.md

```
@@ -0,0 +1,153 @@
+
+## Implementation
+
+### Components Affected
+
+1. **Backend (fragrance-api)**: FastAPI with SQLAlchemy ORM, Alembic migrations
+2. **Frontend (fragrance-ui)**: React with Vite, TypeScript, Tailwind CSS
+3. **Database (fragrance-db)**: PostgreSQL 16 with persistent volume
+
+### Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)